src/playlist-loader.js

```diff
@@ -165,6 +169,28 @@ export const refreshDelay = (media, update) => {
   return delay;
 };
 
+/*
+ * Returns whether the current playlist is the lowest rendition
+ *
+ * @return {Boolean} true if on lowest rendition
+ */
+export const isLowestEnabledRendition = (master, media) => {
```

Comment:
I feel like this would make more sense being in the playlist.js file

Reply:
Good call. Looks much nicer there 👍